client/src/models/ConditionTreeItem.ts

```diff
@@ -0,0 +1,26 @@
+import { join } from 'path';
+import { TreeItem, TreeItemCollapsibleState } from 'vscode';
+
+class ConditionTreeItem extends TreeItem {
+  constructor (
+    public isMet: boolean,
+    public label: string,
+    public rootPath: string
+  ) {
+    super(label, TreeItemCollapsibleState.None);
+  }
+
+  get description (): string {
+    return this.isMet ? '' : 'Not met';
+  }
+
+  get tooltip (): string {
+    return this.label;
+  }
+
+  get iconPath (): string {
+    return join(this.rootPath, 'client', 'res', 'icons', `${this.isMet ? 'tracked' : 'notTracked'}.svg`);
```

Comment:
I would recommend taking the `${this.isMet ? 'tracked' : 'notTracked'}.svg` and putting it into a variable for readability.

Reply:
Resolved via https://github.com/Microsoft/vscode-mezzurite/pull/33/commits/808368aeed44ebbcd418fe05e9542784865a8bc1.